vite-tests/button.test.js

```diff
@@ -0,0 +1,14 @@
+/* global render */
+// @flow
+import * as React from "react";
+
+import Button from "../packages/wonder-blocks-button/components/button.js";
+
+describe("foo", () => {
+    test("it should render a button", async () => {
+        // $FlowFixMe
+        const element = await render(<Button>Hello, world!</Button>);
+        const text = await element.getText();
+        expect(text).toBe("Hello, world!");
+    });
+});
```

Comment:
I'd like these test files to live in the same hierarchy as the source files.  In order to differentiate them from other test files I was thinking of having a structure like:
```
- wonder-blocks-button
-- components
--- __test__ // contains regular jest tests
---- button.test.js
--- __vite__ // contains vite tests
---- button.test.js
--- button.js
--- button-core.js
```
another option is
```
- wonder-blocks-button
-- components
--- __test__
---- button.test.js
---- button.vite_test.js // or just button.vite.js
--- button.js
--- button-core.js
```